jina/parsers/orchestrate/pod.py

```diff
@@ -280,3 +281,9 @@ def mixin_stateful_parser(parser):
         nargs='+',
     )
 
+    gp.add_argument(
```

Comment:
this should be at Deployment level, not Pod level

Reply:
actually, there is a specific section whrre u will find raft related args

Reply:
Look for `mixin_raft_parser` in https://github.com/jina-ai/jina/blob/defd001ab257ac16c7e8b38113d3a4bffd5d226b/jina/parsers/orchestrate/runtimes/runtime.py#L27